packages/experimental/AppearanceAdditions/src/NativeAppearanceAdditions.types.ts

```diff
@@ -1,9 +1,11 @@
 export interface AppearanceAdditions {
-  readonly horizontalSizeClass: SizeClass;
-  readonly userInterfaceLevel: UserInterfaceLevel;
-  readonly accessibilityContrastOption: AccessibilityContrastOption;
+  horizontalSizeClassForRootTag(rootTag: number): SizeClass;
+  userInterfaceLevelForRootTag(rootTag: number): UserInterfaceLevel;
+  accessibilityContrastOptionForRootTag(rootTag: number): AccessibilityContrastOption;
```

Comment:
The docs say not to rely on Root tag being a number. Can we use a different type here? Something more specific that comes from the RN Types? 
https://reactnative.dev/docs/roottag#future-plans

Reply:
Yeah I had put this as a concern in the description. Can you explain more what you mean by "something more specific that comes from the RN Types"? I had actually thought I would need to make the type more generic

Reply:
Use the `RootTag` type from Typescript: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/Libraries/ReactNative/RootTag.d.ts

Right now that's just a number... but in the future if that becomes, let's say.. a string, then our JS code would still run.

Reply:
I also left a comment of what we can use inside our Objective-C type for the same purpose, in the absence of a formal type from RNCore (`id<NSCopying>`).

Reply:
Thanks Saad - didn't realize there was type in React Native I could use. Just updated to use `import type { RootTag } from 'react-native';` - let me know if this is what you were thinking. The link you posted is from "DefinitelyTyped" - were you expecting the import to be from that?

@mischreiber I tried id<NSCopying> but got this error when I tried to use it, since reactTag is strongly typed as an NSNumber. 

![image](https://user-images.githubusercontent.com/78454019/223855779-b33c74a0-37cc-452c-b0a6-26ed850c3c67.png)